src/GovernorCountingFractional.sol

```diff
@@ -38,6 +39,14 @@ abstract contract GovernorCountingFractional is Governor {
      */
     mapping(uint256 => mapping(address => uint128)) private _proposalVotersWeightCast;
 
+    /**
+     * @dev Mapping from voter address to signature-based vote nonce. The
+     * voter's nonce increments each time a signature-based vote is cast with
+     * fractional voting params and must be included in the `params` as the last
+     * 32 bytes when signing for a fractional vote.
+     */
+    mapping(address => uint128) public nonces;
```

Comment:
RR I think we should consider a more descriptive name here. I also wonder if this should be singular, instead of plural, because of how it will look at the callsite for an integrator. So for example, a name like `nextFractionalVoteNonce` or something similar.

To demonstrate what it would read like for an integrator:

```solidity
uint128 _nonce = governor.nonces(my_address);
```

vs.

```solidity
uint128 _nonce = governor.nextFractionalVoteNonce(my_address);
```

What do you think?

Reply:
oh good point I agree it should be singular, but I'm indifferent about naming the var `nonce` vs. `nextFractionalVoteNonce` or `fractionalVoteNonce`

I think "next" is a bit ambiguous since this is really the current nonce you are supposed to use, whereas "next" can be interpreted as "use this next" or "what the next one will be after you use the current"

Reply:
Fair point on "next". I'm fine with `fractionalVoteNonce` if @davidlaprade agrees.

Reply:
I enjoy named mappings, but I know OZ doesn't use them.

Something like:
`mapping(address signer => uint128 nonce) public nonces;`

Reply:
Went with `fractionalVoteNonce`, good suggestion! Done in a61d7b9

Reply:
lol ngl I didn't even know these were a thing @jonwalch!!! 🤯 

Anyway I like the change @davidlaprade and probably best to stick with the OZ convention.

Reply:
Named mappings we’re introduced in 0.8.18 so we’d have to change the pragma anyway, which we should avoid (even though I also prefer named mappings)

Reply:
Maybe too bleeding edge, but they bumped pragmas on master 4 days ago.

https://github.com/OpenZeppelin/openzeppelin-contracts/commit/7bb5592ad5f7a1f28efbc358940f251ad0841366

Reply:
Will look forward to using these in the future but I think too fresh for right now. Thanks @jonwalch. Resolving this issue with @davidlaprade's change!